tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -2251,6 +2251,9 @@ void HWCommandQueue::read_completion_queue() {
                                 channel);
                             uint32_t event_completed =
                                 dispatch_cmd_and_event.at(sizeof(CQDispatchCmd) / sizeof(uint32_t));
+                            CQDispatchCmd *cmd = (CQDispatchCmd *) dispatch_cmd_and_event.data();
+                            CQDispatchCmdId cmd_id = cmd->base.cmd_id;
+
```

Comment:
Is this going to be used for something in the future or is it dead code?

Reply:
It's dead code, thanks for catching